client/src/styles/components/bootstrap.modal.css

```diff
@@ -0,0 +1,41 @@
+/* TODO: Migrate any other should-be-global modal styles to here */
```

Comment:
👍  nice

Reply:
Thanks!

I'm hoping this paves the way for others to consolidate modal styles, or add new modal styles in one location. Ideally, the stylesheet actually moves out of global and into a component for [FP-344](https://jira.tacc.utexas.edu/browse/FP-344). Hm, gonna add that note.